src/CommitmentsV2/SFA.DAS.CommitmentsV2/Validators/UlnValidator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FluentValidation;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.Learners.Validators;
+
+namespace SFA.DAS.CommitmentsV2.Validators
+{
+    public class UlnValidator :AbstractValidator<IUln>
+    {
+        public UlnValidator()
+        {
+            RuleFor(model => model.ULN).Must(UlnValidation).WithMessage(x => $"{FailedValidationMessage(x.ULN)}");
+        }
+
+        private bool UlnValidation(string uln)
+        {
+            var validator = new Learners.Validators.UlnValidator();
+            switch (validator.Validate(uln))
+            {
+                case UlnValidationResult.Success:
+                    return true;
+                default:
+                    return false;
+            }
```

Comment:
Could this just be just 

return (validator.Validate(uln) == UlnValidationResult.Success)

Reply:
Indeed it can. Addressed in https://github.com/SkillsFundingAgency/das-commitments/pull/469/commits/8a7d4226ead6b20c30b28560c3eb6f311ab12980